README.aspnet.md

```diff
@@ -116,6 +116,6 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 
 # License
 
-* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
-* The .NET Framework images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
-* The [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/) also applies to the [SDK images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).
+* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to [].NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```

Comment:
The one link is messed up.

Reply:
That's from an old commit.  It's since been fixed.